SIT.Manager/Localization/zh-TW.axaml

```diff
@@ -444,7 +444,7 @@
 	<system:String x:Key="DirectConnectViewEnterPasswordPlaceholder">Enter Password...</system:String>
 	<system:String x:Key="DirectConnectViewRememberMeCheckBoxTitle">Remember Me</system:String>
 	<system:String x:Key="DirectConnectViewConnectButtonTitle">Connect</system:String>
-	<system:String x:Key="CDirectConnectViewonnectButtonTitleToolTip">Connect to the server</system:String>
+	<system:String x:Key="DirectConnectViewonnectButtonTitleToolTip">Connect to the server</system:String>
```

Comment:
I don't know how the C ended up at the front 😆 

Can I be a pain and ask you to make the key DirectConnectViewConnectButtonTitleToolTip and change it in the other localisations and the DirectConnectView please?

Reply:
Sure np

Reply:
eep